### PR TITLE
Increasing test certificate loader mutex timeout.

### LINKED ISSUE #19430: SslStream test failing to access Mutex
@cipop, this looks like fall out from your recent change:
https://ci.dot.net/job/dotnet_corefx/job/master/job/osx10.12_release_prtest/3093/consoleText
```
System.Net.Security.Tests.SslStreamStreamToStreamTest_Async.SslStream_StreamToStream_Successive_ClientWrite_Sync_Success [FAIL]
        Cannot acquire the global certificate mutex.
        Expected: True
        Actual:   False
        Stack Trace:
           /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx10.12_release_prtest/src/Common/tests/System/Net/Configuration.Certificates.cs(38,0): at System.Net.Test.Common.Configuration.Certificates.GetCertificateCollection(String certificateFileName)
           /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx10.12_release_prtest/src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs(391,0): at System.Net.Security.Tests.SslStreamStreamToStreamTest_Async.DoHandshake(SslStream clientSslStream, SslStream serverSslStream)
           /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx10.12_release_prtest/src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs(78,0): at System.Net.Security.Tests.SslStreamStreamToStreamTest.SslStream_StreamToStream_Successive_ClientWrite_Sync_Success()
     System.Net.Security.Tests.SslStreamStreamToStreamTest_BeginEnd.SslStream_StreamToStream_Authentication_Success [FAIL]
        Cannot acquire the global certificate mutex.
        Expected: True
        Actual:   False
        Stack Trace:
           /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx10.12_release_prtest/src/Common/tests/System/Net/Configuration.Certificates.cs(38,0): at System.Net.Test.Common.Configuration.Certificates.GetCertificateCollection(String certificateFileName)
           /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx10.12_release_prtest/src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs(404,0): at System.Net.Security.Tests.SslStreamStreamToStreamTest_BeginEnd.DoHandshake(SslStream clientSslStream, SslStream serverSslStream)
           /Users/dotnet-bot/j/workspace/dotnet_corefx/master/osx10.12_release_prtest/src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs(36,0): at System.Net.Security.Tests.SslStreamStreamToStreamTest.SslStream_StreamToStream_Authentication_Success()
```

### PR DESCRIPTION
**Pull request by CIPop**
Fixes #19430.

### CODE REVIEW

File: src/Common/tests/System/Net/Configuration.Certificates.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static partial class Certificates
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
             private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
-            private const int MutexTimeout = 5 * 1000;
+            private const int MutexTimeout = 30 * 1000;
```
Comment (davidsh) [inline src/Common/tests/System/Net/Configuration.Certificates.cs]: nit:  a comment at the end of this line like "// 30 seconds" would be useful.  It isn't obvious that the units are milliseconds.

**Comment by stephentoub** [comment]
Merging as several legs have been failing

File: src/Common/tests/System/Net/Configuration.Certificates.cs (hunk 1 above)
Comment (CIPop) [inline src/Common/tests/System/Net/Configuration.Certificates.cs]: Thanks @davidsh: I'll change this next time - there are many places that use this notation and I thought that writing it this way is sufficient.

**Comment by CIPop** [comment]
Thanks @davidsh, @stephentoub !

**Pull request merged**